src/styles/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 4% auto;
+  width: $calendar-width;
+  gap: $gap-size;
+
+  @each $day, $number in $week {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($width + $gap-size) * 6;
```

Comment:
You have an error here

That resulted in the wrong calendar display for the different start day
Namely styles `calendar--start-day--mon`, `calendar--start-day--tue`, etc don't work corerctly